.github/workflows/ci.yml

```diff
@@ -32,6 +32,8 @@ jobs:
         include:
           - container_image: registry.fedoraproject.org/fedora:38
             dotnet_version: "8.0"
+          - container_image: registry.fedoraproject.org/fedora:39
```

Comment:
.NET 8 should be available everywhere now, so maybe we can remove all these items under `include` and just add `8.0` to `dotnet_versions`?

Reply:
> .NET 8 should be available everywhere now, so maybe we can remove all these items under `include` and just add `8.0` to `dotnet_versions`?

Sounds good, I've just updated it.